wagtail/admin/api/actions/copy_for_translation.py

```diff
@@ -0,0 +1,49 @@
+from django.core.exceptions import ValidationError as DjangoValidationError
+from django.shortcuts import get_object_or_404
+from rest_framework import fields, status
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.serializers import Serializer
+
+from wagtail.api.v2.utils import BadRequestError
+from wagtail.core.actions.copy_for_translation import (
+    CopyForTranslationAction, ParentNotTranslatedError)
+from wagtail.core.models.i18n import Locale
+
+from .base import APIAction
+
+
+class CopyForTranslationAPIActionSerializer(Serializer):
+    locale = fields.CharField(max_length=100)
+    copy_parents = fields.BooleanField(default=False, required=False)
+    alias = fields.BooleanField(default=False, required=False)
+    include_subtree = fields.BooleanField(default=False, required=False)
```

Comment:
I think for consistency with copy, we should rename this to ``recursive``: https://github.com/wagtail/wagtail/blob/main/wagtail/admin/api/actions/copy.py#L19